### PR TITLE
DT-2295 bean ordering

### PR DESCRIPTION
**Pull request by mikehalmamoj**
As our library's auto-configuration produces HealthIndicator beans, it must run before the health auto-configuration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -192,7 +192,7 @@ As each topic has its own access key and secret we create an Amazon SNS client f
 
 To remove this pain each topic defined in `HmppsSqsProperties` should have an `AmazonSNS` created.
 
-The bean names have the format `<queueId-sns-client` and can be used with `@Qualifier` to inject the beans into another `@Component`:
+The bean names have the format `<topicId>-sns-client` and can be used with `@Qualifier` to inject the beans into another `@Component`:
```
Comment (mikehalmamoj) [inline README.md:195]: Unrelated typo fix

**Review by petergphillips — APPROVED** [review]
LGTM
